clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/config/GoogleInfrastructureProviderConfig.groovy

```diff
@@ -109,6 +110,11 @@ class GoogleInfrastructureProviderConfig {
                                                           credentials,
                                                           objectMapper)
 
+        newlyAddedAgents << new GoogleHttpLoadBalancerCachingAgent(googleConfiguration.googleApplicationName(),
```

Comment:
Have you tested and confirmed this works when the project has 0 HttpLoadBalancing elements?